resources/language/en_us.ini

```diff
@@ -0,0 +1,1565 @@
+visit.noArguments.teleportError = "§cThere was an error while teleporting you to plot (§6{%1}§c;§6{%2}§c)§8: §7{%0}"
+; this message gets displayed when the player was successfully teleported to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+visit.noArguments.success = "§7You were teleported to plot (§6{%1}§c;§6{%2}§c)§8: §7{%0}"
+; this message gets displayed when the command executor has no plots claimed
+visit.oneArgument.sender.noPlots = "§cYou don't own any plots."
+; this message gets displayed when the command executor has no plot with the provided number
+; {%0} = the provided plot number
+visit.oneArgument.sender.noPlot = "§cYou don't own plot §6{%0}§c."
+; this message gets displayed when there was an error during teleporting the command executor to the plot
+; {%0} = the provided plot number
+; {%1} = the world name of the plot
+; {%2} = the x coordinate of the plot
+; {%3} = the z coordinate of the plot
+visit.oneArgument.sender.teleportError = "§cThere was an error while teleporting you to plot (§6{%2}§c;§6{%3}§c)§8: §7{%0}"
+; this message gets displayed when the command executor was successfully teleported to the plot
+; {%0} = the provided plot number
+; {%1} = the world name of the plot
+; {%2} = the x coordinate of the plot
+; {%3} = the z coordinate of the plot
+visit.oneArgument.sender.success = "§7You were teleported to plot (§6{%2}§c;§6{%3}§c)§8: §7{%0}"
+; this message gets displayed when the provided player has no plots claimed
+; {%3} = the provided players name
+visit.oneArgument.player.noPlots = "§6{%0} §cdoesn't own any plots."
+; this message gets displayed when there was an error during teleporting the command executor to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+; {%3} = the provided players name
+visit.oneArgument.player.teleportError = "§cThere was an error while teleporting you to plot (§6{%1}§c;§6{%2}§c)§8: §7{%0}"
+; this message gets displayed when the player was successfully teleported to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+; {%3} = the provided players name
+visit.oneArgument.player.success = "§7You were teleported to plot (§6{%1}§c;§6{%2}§c)§8: §7{%0}"
+; this message gets displayed when there couldn't be found a plot with the provided plot alias
+; {%0} = the provided plot alias
+visit.oneArgument.alias.noPlot = "§cThere is no plot with alias §6{%0}§c."
+; this message gets displayed when there was an error during teleporting the command executor to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+; {%3} = the provided plot alias
+visit.oneArgument.alias.teleportError = "§cThere was an error while teleporting you to plot (§6{%2}§c;§6{%3}§c)§8: §7{%0}"
```

Comment:
Maybe `§cThere was an error while teleporting you to the plot (§6{%1}§c;§6{%2}§c) §cwith alias §6{%3}§c.`